src/transformers/modeling_longformer.py

```diff
@@ -476,25 +476,18 @@ def __init__(self, config):
         super().__init__(config)
 
         if isinstance(config.attention_window, int):
-            assert config.attention_window % 2 == 0, "`attention_window` has to be an even value"
-            assert config.attention_window > 0, "`attention_window` has to be positive"
+            assert config.attention_window % 2 == 0, "`config.attention_window` has to be an even value"
+            assert config.attention_window > 0, "`config.attention_window` has to be positive"
             config.attention_window = [config.attention_window] * config.num_hidden_layers  # one value per layer
         else:
             assert len(config.attention_window) == config.num_hidden_layers, (
-                "`len(attention_window)` should equal `num_hidden_layers`. "
+                "`len(config.attention_window)` should equal `config.num_hidden_layers`. "
                 f"Expected {config.num_hidden_layers}, given {len(config.attention_window)}"
             )
 
-        if config.attention_mode == "bert":
```

Comment:
@thomwolf @sshleifer @ibeltagy - I removed the `attention_mode == bert` option here. As explained above, I don't think we should keep this.

Reply:
I am ok with this.